sync/src/types.rs

```diff
@@ -122,8 +124,8 @@ impl PeerState {
     }
 
     pub fn can_sync(&self, now: u64, ibd: bool) -> bool {
-        // only sync with outbound peer in IBD
-        (self.is_outbound || !ibd)
+        // only sync with outbound/whitelist peer in IBD
+        ((self.is_outbound || self.chain_sync.is_whitelist) || !ibd)
```

Comment:
redundant brackets

Reply:
I think this bracket can help understand